ChronoAPI/ChronoLog/include/ChronicleMetadataRPCVisor.h

```diff
@@ -98,97 +79,60 @@ class ChronicleMetadataRPCProxy {
         return g_chronicleMetaDirectory->create_story(chronicle_name, story_name, attrs);
     }
 
-    bool CreateStory(std::string &chronicle_name, std::string &story_name,
-                     const std::unordered_map<std::string, std::string> &attrs) {
-        LOGD("%s is called in PID=%d, with args: chronicle_name=%s, story_name=%s, ,attrs=",
-             __FUNCTION__, getpid(), chronicle_name.c_str(), story_name.c_str());
-        for (auto iter = attrs.begin(); iter != attrs.end(); ++iter) {
-            LOGD("%s=%s", iter->first.c_str(), iter->second.c_str());
-        }
-        return CHRONOLOG_RPC_CALL_WRAPPER("CreateStory", 0, bool, chronicle_name, story_name, attrs);
-    }
-
     bool LocalDestroyStory(std::string &chronicle_name, std::string &story_name, const int &flags) {
         LOGD("%s is called in PID=%d, with args: chronicle_name=%s, story_name=%s, flags=%d",
              __FUNCTION__, getpid(), chronicle_name.c_str(), story_name.c_str(), flags);
         extern std::shared_ptr<ChronicleMetaDirectory> g_chronicleMetaDirectory;
```

Comment:
It may be wise to add some basic error checks in all functions that access the metadata data structure. Check for input data integrity.

Reply:
Good idea. I will add that.